### PR TITLE
new formulation of the capacity maintenance

### PR DESCRIPTION
**Pull request by ClaraLuisa**
The capacity maintenance was seperated into three seperate equations to ensure that capacity that is built during one periode also exists as cap by the end of the periode.

### CODE REVIEW

**Comment by ClaraLuisa** [comment]
I am rather indifferent about naming the equations, but I guess naming it _NEW might be a smart idea.

File: message_ix/model/MESSAGE/model_core.gms [hunk 1]
```diff
@@ -248,7 +248,9 @@ Equations
     COMMODITY_BALANCE               commodity supply-demand balance constraint
     STOCKS_BALANCE                  commodity inter-temporal balance of stocks
     CAPACITY_CONSTRAINT             capacity constraint for technology (by sub-annual time slice)
-    CAPACITY_MAINTENANCE            constraint for technology capacity maintainance
+    CAPACITY_MAINTENANCE_HIST       constraint for capactiy maintainance  historical installation (built before start of model horizon)
+    CAPACITY_MAINTENANCE_CURR       constraint for capactiy maintainance of new capacity built in the current period (vintage == year)
+    CAPACITY_MAINTENANCE            constraint for capacity maintainance (at the end of previous periode)
```
Comment (danielhuppmann) [inline message_ix/model/MESSAGE/model_core.gms]: there is a typo here ('periode'), and I would rather call that text "constraint for capacity maintainance over the technical lifetime"

**Comment by danielhuppmann** [comment]
lgtm, thanks @ClaraLuisa - can you please incorporate these two comments and also rewrite the LaTeX documentation to reflect that this is now formulated as an equality in the first period

**Comment by ClaraLuisa** [comment]
@gidden should I rename and commit again?

**Comment by gidden** [comment]
If it matches to the CAP_NEW variable, then I would suggest yes

**Comment by ClaraLuisa** [comment]
Sure @danielhuppmann and @gidden. However, my spelling and grammar aren't the best. But I can definitely try.

**Comment by danielhuppmann** [comment]
one final request, @ClaraLuisa - can you please add a line to the release notes, something like "reformulated capacity maintainance constraint to ensure that newly installed capacity cannot be decommissioned immediately" (or in any case something more intuitive than "exists as cap by the end of the period").

**Comment by gidden** [comment]
rebased onto hackathon-prs to keep everything in one simply pull when we're ready! once this passes CI I will merge in

**Comment by gidden** [comment]
hey @ClaraLuisa, do you mind if I rebase this and add it to our hackathon-prs branch?

**Comment by ClaraLuisa** [comment]
hey @gidden, rebasing is no problem for me, however, I created the branche here as @danielhuppmann asked me to do so. So maybe he has something to add to this?

**Comment by danielhuppmann** [comment]
no concerns from my side!

**Comment by gidden** [comment]
closing in favor of #96

**Pull request closed without merging**